cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
+				if name == "" && n.Description != nil {
+					name = n.Description.Hostname
```

Comment:
Is searching by hostname something that we did before, or is this a new feature?